src/services/ImageRepository.cpp

```diff
@@ -10,8 +10,8 @@
 namespace ra {
 namespace services {
 
-constexpr char* DefaultBadge = "00000";
-constexpr char* DefaultUserPic = "_User";
+inline constexpr const char* DefaultBadge{ "00000" };
```

Comment:
`Error C2440 'initializing': cannot convert from 'const char [6]' to 'char* const '`


I could have just used the macro I put a while back; `_CONSTANT_VAR ` ...
- Expands to` inline constexpr auto`, where `auto `is `const char* const`), 
- The `const `at the end usually doesn't show up in intellisense

..but felt it would be better do it this way to show people how to do it explicitly.
The inline is a performance boost (still correct) but the `const `part is what was needed because the left side was` char*` and the right side was `char* const` (const pointer)